runtime/src/bank.rs

```diff
@@ -1530,8 +1530,8 @@ impl Bank {
         self.epoch_schedule.get_epoch_and_slot_index(slot)
     }
 
-    pub fn is_votable(&self) -> bool {
-        self.is_delta.load(Ordering::Relaxed) && self.tick_height() == self.max_tick_height
+    pub fn is_empty(&self) -> bool {
+        self.is_delta.load(Ordering::Relaxed)
```

Comment:
is_delta == is_empty? Confusing terminology. Anything you can do here? is "delta" the right word?

Reply:
hehe b/c I messed up, it should be`self.is_delta.load(Ordering::Relaxed)`, does that make more sense?

Reply:
Makes a little more sense.  A bit disturbing though. Did you updated the tests to reflect the implementation rather than updating the implementation to reflect better tests?

Reply:
Even worse, I updated the tests and assumed it would work, thinking that `is_empty == is_votable` which is complete nonsense 😛 

Updated the test, thanks!